site/_posts/2016-11-21-last-week-in-manageiq-euwe_stabilization_continues.html.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Last week in ManageIQ – Euwe stabilization continues
+author: Laura
+date: 2016-11-21
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi! This is [Laura](https://github.com/lgalis) with some highlights of our Last Week in ManageIQ.
+
+## Featured
+
+The stabilization of the upcoming Euwe release was the main focus for us last week. There were 108 [PRs] merged this week, containing 226 [commits] by 52 [contributors].
+
+If you would like to contribute, you can choose from our list of [open issues].
+
+Sprint 49 ended on Monday, November 14th. The sprint review meeting has been recorded, as always, and can be found [here](https://www.youtube.com/watch?v=Y4dU1fmRgX4).
```

Comment:
Do we have an existing (or about to exist) blog post about the spring review? Do we want to mention it again here? /cc @cybette